recordedit/recordEdit.app.js

```diff
@@ -253,14 +253,22 @@
                                 throw new Errors.noRecordError({}, page.reference.displayname.value, recordSetLink);
                             }
 
-                            var column, value;
-
+                            var column, value, headTitle;
 
                             // $rootScope.tuples is used for keeping track of changes in the tuple data before it is submitted for update
                             $rootScope.tuples = [];
-                            if (context.mode == context.modes.EDIT && page.tuples.length == 1) {
-                                $rootScope.displayname = page.tuples[0].displayname;
+                            if (context.mode == context.modes.EDIT) {
```

Comment:
Looking at the code and the way you implemented it, I realized we never considered the copy case.

It seems like in case of copy we use the same page title as create mode (`Create new <table>`) which I think makes sense. So I think you should  also do the same for the head title.

Reply:
Oh that's a good call. I'll update that as well and add a test case for it.